front/app/components/AvatarBubbles/index.tsx

```diff
@@ -11,38 +11,10 @@ import useRandomAvatars from 'api/avatars/useRandomAvatars';
 import { ScreenReaderOnly } from 'utils/a11y';
 import { useIntl } from 'utils/cl-intl';
 
+import { OuterContainer, BubbleContainer } from './Containers';
 import messages from './messages';
 import placeholderImage from './user.png';
-
-const getFontSize = (size: number, digits: number) => {
-  if (size >= 34) {
-    if (digits <= 2) {
-      return 14;
-    }
-
-    if (digits === 3) {
-      return 12;
-    }
-
-    if (digits >= 4) {
-      return 11;
-    }
-  } else {
-    if (digits <= 2) {
-      return 12;
-    }
-
-    if (digits === 3) {
-      return 11;
-    }
-
-    if (digits >= 4) {
-      return 10;
-    }
-  }
-
-  return 14;
-};
+import { getFontSize } from './utils';
```

Comment:
Refactored this component a bit to make certain parts of it reusable. And then reused those in the skeleton.